org.scala-ide.sdt.core/src/org/scalaide/ui/editor/DefaultScalaEditorConfiguration.scala

```diff
@@ -0,0 +1,104 @@
+package org.scalaide.ui.editor
+
+import org.eclipse.jface.text.source.IAnnotationHover
+import org.eclipse.jface.preference.IPreferenceStore
+import org.eclipse.jface.text.reconciler.IReconciler
+import org.eclipse.jface.text.reconciler.MonoReconciler
+import org.eclipse.jface.text.source.DefaultAnnotationHover
+import org.scalaide.core.lexical.ScalaPartitions
+import org.eclipse.jface.text.presentation.PresentationReconciler
+import org.scalaide.core.lexical.ScalaCodeScanners
+import org.eclipse.jface.text.source.SourceViewerConfiguration
+import org.eclipse.jface.text.information.InformationPresenter
+import org.eclipse.jdt.ui.text.IJavaPartitions
+import org.eclipse.jface.text.IDocumentListener
+import org.eclipse.jface.text.source.ISourceViewer
+import org.eclipse.jface.text.IDocument
+import org.scalaide.ui.editor.hover.IScalaHover
+import org.eclipse.jface.text.ITextHover
+import org.scalaide.core.IScalaPlugin
+import org.eclipse.jface.text.DocumentEvent
+import org.eclipse.jface.util.PropertyChangeEvent
+import org.eclipse.jface.text.rules.DefaultDamagerRepairer
```

Comment:
What is the official standpoint for imports in codebase? Should we organize them all?